docs/content/reference/versioning-support.md

```diff
@@ -1,42 +1,75 @@
 
-- API `hypershift.openshift.io`.
-- HyperShift Operator (HO).
-- Control Plane Operator (CPO).
-- HyperShift CLI.
+- Management Cluster
+- API `hypershift.openshift.io`
+- HyperShift Operator (HO)
+- Control Plane Operator (CPO)
+- HyperShift CLI
+- Hosted Control Plane (HCP) CLI
+
+## Support Level
+### Management Cluster
+There is no hard requirement on what OCP version the OpenShift management cluster needs to be running.
+
+The OCP versions of HostedClusters that can be installed on the management cluster depend on the HO version installed.
 
-## Support level
+### API
+There are two user facing resources exposed by HyperShift: [HostedClusters and NodePools](https://hypershift-docs.netlify.app/reference/api/).
+
+The HyperShift API version policy generally aligns with the [Kubernetes API versioning](https://kubernetes.io/docs/reference/using-api/#api-versioning).
 
 ### HO
+The HO is not tied to a particular OCP release. It is tied to the `main` branch of the HyperShift repo regardless of CPO version.
+
+The HO is tagged at particular commits as part of merging new HO versions for Red Hat managed services; there is no particular tagging scheme for this effort.
 
-- A HO is intended to be released for each OCP release.
-- A HO released for OCP N minor release must support N, N-1 and N-2 OCP minor releases.
-- A HO release if anything, will only do best effort to support future OCP versions.
-- The HO, once installed, will create a ConfigMap called `supported-versions` into the Hypershift namespace which describes the HostedClusters supported versions that could be deployed. This is how it looks like:
-    ```
-    apiVersion: v1
-    data:
-      supported-versions: '{"versions":["4.13","4.12","4.11"]}'
-    kind: ConfigMap
-    metadata:
-      labels:
+A list of the tags can be found [here](https://github.com/openshift/hypershift/tags).
+
+Once installed, the HO creates a ConfigMap called `supported-versions` into the Hypershift namespace, which describes the HostedClusters supported versions that could be deployed. 
+
+Here is an example `supported-versions` ConfigMap:
+```
+apiVersion: v1
+data:
+    server-version: 2f6cfe21a0861dea3130f3bed0d3ae5553b8c28b
+    supported-versions: '{"versions":["4.17","4.16","4.15","4.14"]}'
+kind: ConfigMap
+metadata:
+    creationTimestamp: "2024-06-20T07:12:31Z"
+    labels:
         hypershift.openshift.io/supported-versions: "true"
-      name: supported-versions
-      namespace: hypershift
-    ```
+    name: supported-versions
+    namespace: hypershift
+    resourceVersion: "927029"
+    uid: f6336f91-33d3-472d-b747-94abae725f70
+```
+
+!!! important
+
+        You cannot install HCs higher than what the HO supports. In the example above, HCs using images greater than 4.17 cannot be created.
 
 ### CPO
+The CPO is released as part of each OCP payload release image. You can find those release images here:
```

Comment:
Documentation on the CPO and how it supports or relates to the support matrix?

Reply:
Not sure which support matrix you're referencing? FWIW there is no direct support matrix between hub (management) OCP and OCP in hosted clusters.